### ISSUE TITLE
Add in documentation that Node.js global vars may be keeped between deployments

### ISSUE THREAD

**Issue by yvele** [Azure/azure-webjobs-sdk-script#1234]
Specially usefull information when using Webpack to [drastically reduce Azure Function cold start](https://github.com/Azure/azure-webjobs-sdk-script/issues/298#issuecomment-281779696).

I understand the global vars sharing when I tried to use `babel-polyfill` and got this error:

> Error: only one instance of babel-polyfill is allowed

[Babel polyfill](https://babeljs.io/docs/usage/polyfill/) is using global vars and although I was importing it only once (in my `webpack.config`), I constantly get this error, and I finally had to make a little wrapper around my Azure Function handle to make sure `babel-polyfill` is not already loaded in global vars before requiring it.

I suggest adding a global vars warning in the documentation (especially with all Node.js developers going to make a webpack bundle to [reduce Azure Function cold start](https://github.com/Azure/azure-webjobs-sdk-script/issues/298#issuecomment-281779696).

This issue is kind of related to https://github.com/Azure/azure-webjobs-sdk-script/issues/298#issuecomment-281779696 and attempts to make Azure Function work with Webpack.

**Comment by lindydonna** [comment]
@christopheranderson @yochay Shouldn't this be moved to the Functions repo, since it's about documentation?

**Comment by christopheranderson** [comment]
@lindydonna - yes this should be moved

**Comment by lindydonna** [comment]
@christopheranderson Could you file a new issue in the docs repo to track this?  You have more context on the doc requirement.

**Comment by paulbatum** [comment]
This issue was moved to Azure/Azure-Functions#368
